src/app/components/share-content/share-content.component.ts

```diff
@@ -30,10 +30,10 @@ import { ThemeService } from '../../providers/theme.service';
 	`,
 })
 export class ShareContentComponent {
-	route = input<string>('');
-	params = input<{ [key: string]: string }>({});
-	message = input<string>('');
-	isLoading = input<boolean>(false);
+	@Input() route: string = '';
```

Comment:
Something's wrong here, somehow these changes are overriding previous changes. You may want to sync with what's in dev on your fork.